packages/auth/src/providers/cognito/apis/confirmUserAttribute.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { AuthValidationErrorCode } from '../../../errors/types/validation';
+import { assertValidationError } from '../../../errors/utils/assertValidationError';
+import { ConfirmUserAttributeRequest } from '../../../types/requests';
+import { verifyUserAttribute } from '../utils/clients/CognitoIdentityProvider';
+import { VerifyUserAttributeException } from '../types/errors';
+import {
+	AmplifyV6 as Amplify,
+	assertTokenProviderConfig,
+} from '@aws-amplify/core';
+import { fetchAuthSession } from '../../../';
+import { getRegion } from '../utils/clients/CognitoIdentityProvider/utils';
+import { assertAuthTokens } from '../utils/types';
+import { CognitoUserAttributeKey } from '../types';
+
+/**
+ * Confirm the user attribute with the confirmation code.
+ *
+ * @param confirmUserAttributeRequest - The ConfirmUserAttributeRequest
+ *
+ * @throws  -{@link AuthValidationErrorCode } -
+ * Thrown when `confirmationCode` is not defined.
+ *
+ * @throws  -{@link VerifyUserAttributeException }
```

Comment:
Should this have a description?

```suggestion
 * @throws  -{@link VerifyUserAttributeException }
```